### ISSUE TITLE
External links routed through CloudFlare network can cause an error

### ISSUE THREAD

**Issue by goslingo** [vegaprotocol/vega-website-cypress-tests#11]
The following link causes the **"check all links in main"** test to fail:

https://www.tandfonline.com/doi/pdf/10.1080/1350486X.2022.2030773?needAccess=true

```
The response we received from your web server was:
  > 503: Service Unavailable
This was considered a failure because the status code was not `2xx` or `3xx`.
If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`
```

When visiting the link it first goes to an interim CloudFlare page with a 503 http response, this is presumably for security reasons, e.g. DDOS protection.

Normal visitors to this page will then be seamlessly redirected to the PDF, unless they have to undertake a captcha-like challenge first, however cypress will stop here and rightly throw an error.

We can turn off the failOnStatusCode setting which would bypass this problem, but then our tests wouldn't pick up broken links, so it's not a solution.

**Comment by david-boydell** [comment]
Hi Oli,

I haven't had a lot of luck with this I'm afraid. I read that Cloudflare does a particular check against the headers used in any request and rejects bots etc. but amending the headers of the request made dfid not alter the 503 response.

I have added a temporary fix PR that just skips that url for now. It's not ideal but at least the tests will run green and it'll be a known issue.

Longer term I am looking at using version 10 of Cypress. This allows you to visit different top level domains using a `cy.origin` request - so we can selectively `cy.visit` problematic domains - but I'm struggling to get it to work. I have reached out to the community as this is an experimental feature, hopefully I will hear back soon.
